src/components/Port.vue

```diff
@@ -157,7 +157,7 @@ export default {
       this.loading = true
       axios
         .get(
-          `https://frozen-falls-98471.herokuapp.com/api/v1/port/${
+          `http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com/api/v1/port/${
```

Comment:
please use plugin configurationfor this!
Those are configurable in master.cfg, and available in config dependency injection.

see ex:
https://github.com/buildbot/buildbot/blob/master/www/wsgi_dashboards/src/module/dashboards.route.js#L10

Reply:
Okay, sure. I will actually remove this (it would use the virtual builders-which would be the ports)

Reply:
Yes, this needs to be removed. I consider this to be a quick hack which served the demonstration purposes, but the same needs to be achieved without external help, all the data needs to come from internal data from the buildbot.